### ISSUE TITLE
Inline plotting

### ISSUE THREAD

**Issue by MLKaufman** [quarto-dev/quarto-vscode#61]
Thanks for all the hard work on this plugin!

Feature request:

The ability to have inline plotting below each code block similar to the Jupyter notebook VSCode plugin.

This would enable a more similar preview to the rendered Quarto document, aiding with debugging and formatting.
Additionally this would greatly improve the organization of working in the document, especially with multiple languages, as currently each opens its own interactive side window for plots making it difficult to keep track of which code block matches which generated plot.

**Comment by jjallaire** [comment]
Agreed that this would be a great option. Will keep it in mind as we continue development!

**Comment by kongdd** [comment]
call for this feature too

**Comment by emfeltham** [comment]
this would be really great
